src/main/java/com/backbase/ct/bbfuel/client/user/UserPresentationRestClient.java

```diff
@@ -17,6 +17,7 @@
     private final BbFuelConfiguration config;
 
     private static final String SERVICE_VERSION = "v2";
+    private static final String SERVICE = "client-api";
```

Comment:
I suggest to call this constant `CLIENT_API`

Reply:
done